crates/store/src/nullifier_tree.rs

```diff
@@ -1,7 +1,8 @@
+use miden_crypto::merkle::SMT_DEPTH;
```

Comment:
We should be able to import this from `miden_objects::crypto::merkle` on line 5 below.